### ISSUE TITLE
feat: When a screen-reader uses the akai mpd226 editor, they receive WCAG 4.1.3-compliant status updates

### ISSUE THREAD

**Issue by molenick** [molenick/midilab#64]
After address the major accessibility defects from https://github.com/molenick/midilab/issues/18, the major issue remaining is that screen reader users don't receive proper async updates.

The easy, bad thing is to just "force-focus on update"  but that violates WCAG success criteria 4.1.3.

So the ask here is:
How do we use accesskit and egui to inform screen reader users when such an update happens? Haven't seen it in the docs, but need to take a deeper look.  It's possibly not documented yet.
